src/Markup/Avalonia.Markup.Xaml/Templates/TreeDataTemplate.cs

```diff
@@ -16,7 +18,7 @@ public class TreeDataTemplate : ITreeDataTemplate
         public object Content { get; set; }
 
         [AssignBinding]
-        public Binding ItemsSource { get; set; }
+        public BindingBase ItemsSource { get; set; }
```

Comment:
Why not using `IBinding`?

Reply:
I think BindingBase is the right choice since we only support two specific subtypes and we aren’t using the IBinding interface in the implementation anyway.